src/mixins/userIsTabbing.js

```diff
@@ -0,0 +1,22 @@
+const userIsTabbingMixin = {
+	created: function() {
+		window.addEventListener("keydown", this.handleFirstTab);
+		this.$userIsTabbing = false;
+	},
+	methods: {
+		handleFirstTab: function(e) {
```

Comment:
```suggestion
		$_handleFirstTab: function(e) {
```
to prevent namespace conflicts
https://vuepress.test.schul-cloud.org/0-GettingStarted/3-DirectoryStructure.html#src-mixins